BENEFITS.md

```diff
@@ -0,0 +1,17 @@
+
+## Benefits 
+* Advertisment of offering as "certified"
+* Use of Argo Logo
+* Use of Argo Certified Mark
+* Use of Argo in product naming
+
+## Product Naming
+|  | Today | After Conformance |
+|---|---|---|
+| Acme Cloud for Argo | :white_check_mark: allowed | :white_check_mark: allowed |
+| Acme Platform for Argo | :white_check_mark: allowed | :white_check_mark: allowed |
+| Acme Argo Cloud | :x: not allowed | :x: not allowed |
+| Acme Argo Platform | :x: not allowed | :x: not allowed |
+| Acme Argo CD | :x: not allowed | :white_check_mark: allowed |
```

Comment:
We had the discussion to start with a conservative approach to not allow Argo name including Acme Argo CD or similar, but only give out the certification badge as the stage one benefits target. The idea is to let stage one sort out all the logistics and collect the feedback before we are giving out more marketing and branding things. The concern is the fact that is easy to give out and very hard and time-consuming to take something back. If things are working great with traction and good operational experience. We will work on stage two to give out the additional benefits including Acme Argo CD usage. I believe this approach is more rational and easier to win over the majority of support from all the maintainers to move forward with the conformance plan.

Reply:
@jessesuen to review with @wanghong230 This is ultimately a question of if we believe the requirements are strong enough. I think with @jessesuen 's additions they are but let's discuss and try to resolve in the next week.

Reply:
I feel we are betting on things that will just work magically without gotchas, and won’t take the conservative approach. The benefits are easy to change and update afterward. The hard work is still behind that. What is the process (who, when, how) to enforce the violation? Whether are there bad gaps in our requirements? If we found something, how do we cover that gap and ask everyone to comply with that new requirement? I am not hearing the solid practices and experience about it. I vaguely have the idea that Henrik can help escalate to CNCF's particular email to enforce. I am not convinced about the process, so I believe the benefits are too generous here as the starting point.

Regarding the counter-argument with the examples:
- Kubernetes is the only one that is successful, I am not convinced it is a proper example to follow. It is CNCF’s money printer, so many eyes are on it, to ensure it is on track.
- Prometheus doesn’t give out the naming.
- Crossplane gives out the name, but it is more for the main vendor to allow themself to use it.
- Flux doesn’t work on the conformance for the obvious reason.

@jessesuen @hblixt @todaywasawesome @alexec @wtam2018 @edlee2121